src/components/show-edit-pod/show-edit-pod.stories.js

```diff
@@ -0,0 +1,98 @@
+import { State, Store } from '@sambego/storybook-state';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import ShowEditPod from './show-edit-pod';
+import Content from '../content';
+import Textbox from '../textbox';
+import { notes, info } from './documentation';
+
+const store = new Store({
+  editing: ShowEditPod.defaultProps.editing
+});
+
+const onEdit = () => {
+  store.set({ editing: true });
+  action('edit')();
+};
+
+const onCancel = () => {
+  store.set({ editing: false });
+  action('cancel')();
+};
+
+const onDelete = () => {
+  store.set({ editing: false });
+  action('delete')();
+};
+
+storiesOf('ShowEditPod', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [Content, State]
+    }
+  })
+  .add(
+    'default',
+    () => {
+      const asTheme = select('as', OptionsHelper.themesFull, ShowEditPod.defaultProps.as);
+      const border = boolean('border', ShowEditPod.defaultProps.border);
+      const buttonAlign = select('buttonAlign', OptionsHelper.alignBinary, ShowEditPod.defaultProps.buttonAlign);
+      const cancel = boolean('cancel', ShowEditPod.defaultProps.cancel);
+      const cancelText = cancel ? text('cancelText', ShowEditPod.defaultProps.cancelText) : undefined;
+      const deleteText = text('deleteText', ShowEditPod.defaultProps.deleteText);
+      const editing = boolean('editing', store.get('editing'));
+      const saveText = text('saveText', ShowEditPod.defaultProps.saveText);
+      const saving = boolean('saving', ShowEditPod.defaultProps.saving);
+      const title = text('title', 'Person');
+      const transitionName = text('transitionName', ShowEditPod.defaultProps.transitionName);
+      const validateOnMount = boolean('validateOnMount', ShowEditPod.defaultProps.validateOnMount);
+      const editFields = [
+        <Textbox
+          key='first_name' label='First Name'
+          value='Alan'
+        />,
+        <Textbox
+          key='second_name' label='Second Name'
+          value='Smith'
+        />,
+        <Textbox
+          key='telephone' label='Telephone'
+          value='000 000 0000'
+        />
+      ];
+
+      return (
+        <State store={ store }>
+          <ShowEditPod
+            as={ asTheme }
+            border={ border }
+            buttonAlign={ buttonAlign }
+            cancel={ cancel }
+            cancelText={ cancelText }
+            deleteText={ deleteText }
+            editing={ editing }
+            onDelete={ onDelete }
+            onCancel={ onCancel }
+            onEdit={ onEdit }
+            saveText={ saveText }
+            saving={ saving }
+            title={ title }
+            transitionName={ transitionName }
+            validateOnMount={ validateOnMount }
+            editFields={ editFields }
+          >
+            <Content title='First Name'>Alan</Content>
+            <Content title='Last Name'>Smith</Content>
+            <Content title='Telephone'>000 000 0000</Content>
```

Comment:
![Zrzut ekranu 2019-03-20 o 13 25 40](https://user-images.githubusercontent.com/12142840/54684123-aec33000-4b13-11e9-87c2-6ea7959dfd9e.png)

Reply:
I guess it's the storybook issue